src/script/view_model/CallingViewModel.ts

```diff
@@ -0,0 +1,281 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CALL_TYPE, CONV_TYPE, REASON as CALL_REASON, STATE as CALL_STATE} from '@wireapp/avs';
+import ko from 'knockout';
+import {Logger, getLogger} from 'Util/Logger';
+import {AudioType} from '../audio/AudioType';
+import {Call} from '../calling/Call';
+import {CallingRepository} from '../calling/CallingRepository';
+import {Grid, getGrid} from '../calling/videoGridHandler';
+import {MediaDevicesHandler} from '../media/MediaDevicesHandler';
+import {MediaStreamHandler} from '../media/MediaStreamHandler';
+import {PermissionState} from '../notification/PermissionState';
+
+import '../components/calling/chooseScreen';
+
+declare global {
+  interface HTMLAudioElement {
+    setSinkId: (sinkId: string) => Promise<void>;
+  }
+}
+
+export class CallingViewModel {
+  public readonly audioRepository: any;
+  public readonly callingRepository: CallingRepository;
+  public readonly conversationRepository: any;
```

Comment:
You could  import the repository and specify it here, even when the repository is not converted to TypeScript yet.